### ISSUE TITLE
Not working in 1.0

### ISSUE THREAD

**Issue by sizhky** [zremboldt/obsidian-media-grid#4]
I justs discovered and installed the css file as per instructions, but nothing is happening either in live editor or reader mode. Can anyone confirm that this works in 1.0 ?

**Comment by timmb** [comment]
Also not working for me

**Comment by berot3** [comment]
me too v1.0.3

**Comment by andremeireles** [comment]
Did you guys installed the [contextual typography](https://github.com/mgmeyers/obsidian-contextual-typography) plugin?
It is a requirement for the snippet to work.

**Comment by timmb** [comment]
Yes, contextual typography is installed. I've tested on both MacOS and Windows and no luck. Is it working for you in the latest Obsidian?

**Comment by andremeireles** [comment]
> Yes, contextual typography is installed. I've tested on both MacOS and Windows and no luck. Is it working for you in the latest Obsidian?

It's working for me.
Obsidian 1.0.3. Windows 10.
I tested it on my own vault and on the sandbox vault.
Maybe you forgot to enable the toggle button.

![image](https://user-images.githubusercontent.com/1096931/202539713-1f791583-a9ff-4920-8fbd-5f57a1ad2130.png)

**Comment by timmb** [comment]
Hmm ok on further testing it is working in Reader view, which I thought I'd tested but apparently not.

I guess my issue is now more about Live Preview mode: https://github.com/zremboldt/obsidian-media-grid/issues/1

**Comment by ck7300373** [comment]
@sizhky @timmb @berot3 
Hey guys, there is a workround for OBS 1.0~ :

1. Open the css file with VSCode 

2. Replace ".is-readable-line-width" to "" 
   ( or you can just delete the words ".is-readable-line-width" & all its occurrences. )

3. snippet is working again ~~ !!

**Comment by issam-seghir** [comment]
for me it works in one vault and doesn't work in other vaults, that's strange!!

**Update:** 
I try everything to make this css snippet work in my new vault Until I found this solution

- Make sure you use the pure snippets from this repo without any edits (like replacing ".is-readable-line-width" to "" in the previous comment)
- Copy this setting to `app.josn` in the `.obsidian` folder (make sure you copy all of them even if you activate this setting in the GUI in the obsidian setting)

```json
  "smartIndentList": true,
  "autoPairMarkdown": true,
  "autoPairBrackets": true,
  "foldIndent": true,
  "foldHeading": true,
  "livePreview": true,
  "showIndentGuide": true,
  "lineWrap": true
```
![image](https://user-images.githubusercontent.com/74573675/213141401-d0d62c23-6ae3-4d38-9f71-239b9247cade.png)


This setting i copy from my old vault to my new vault makes the snippet work for me in all themes I have tried

> From AnupPuccin theme 
![image](https://user-images.githubusercontent.com/74573675/213143064-c6ae708f-b289-4a8c-94f8-e42b05d4648c.png)


**My obsidian version: 1.1.8**


**Update 2:** 
After a short time, the snippet became not working, so I don't know if the solution I suggested earlier will work
But after several failed attempts, I finally found the Solution, I don't know if you are facing the same problem but solving this problem made it work for me, and I did not find any problem with the `img-gird` snippet after that, give it a try

https://github.com/whyt-byte/Blue-Topaz_Obsidian-css/issues/505#issuecomment-1404866341

**Comment by HernandoR** [comment]
On 1.1.9 works fine with reading mode, wish for live mode

**Comment by jerviscui** [comment]
> @sizhky @timmb @berot3 Hey guys, there is a workround for OBS 1.0~ :
> 
> 1. Open the css file with VSCode
> 2. Replace ".is-readable-line-width" to ""
>    ( or you can just delete the words ".is-readable-line-width" & all its occurrences. )
> 3. snippet is working again ~~ !!

Great. It's working again.
